test/run_test.py

```diff
@@ -22,7 +22,7 @@
 TESTS = [
     'test_autograd',
     'test_cpp_extensions_aot',
-    'test_cpp_extensions_aot_no_ninja',
+    'test_cpp_extensions_aot_ninja',
```

Comment:
Oh, I see, you renamed test_cpp_extensions_aot -> test_cpp_extensions_aot_ninja and test_cpp_extensions_aot_no_ninja -> test_cpp_extensions_aot.

Could we please have the following instead:
- test_cpp_extensions_aot_ninja runs cpp extensions tests with ninja
- test_cpp_extensions_aot_no_ninja runs cpp extension tests without ninja
So that there is no fishy prefix behavior in run_test.py and to make it clear that the "no ninja build" is not the default one?

Reply:
(resolved)